src/Context.cpp

```diff
@@ -13,10 +13,23 @@
 //
 //===----------------------------------------------------------------------===//
 #include "Context.hpp"
+#include <gtirb/Block.hpp>
+#include <gtirb/DataObject.hpp>
+#include <gtirb/IR.hpp>
+#include <gtirb/ImageByteMap.hpp>
+#include <gtirb/Module.hpp>
 #include <gtirb/Node.hpp>
+#include <gtirb/Section.hpp>
+#include <gtirb/Symbol.hpp>
 
 using namespace gtirb;
 
+// By moving these declarations here, we avoid instantiating the default
```

Comment:
Did you test this with MSVC as well as Clang?

Reply:
Yes.